lando/lando
# 2450/init mm2

Testing:

- `lando init`
  - select `lagoon`
  - enter email address of a valid lagoon instance
  - add the key in the output to your lagoon account
 
- run `lando init` again
  - select `lagoon`
  - select the email address you configured the first time around
  - select the drupal project
  - set a name for your project

This should successfully initialize lando by pulling the `main` branch of the repository and create a `.lando.yml`.
